tests/PHPUnit/Integration/CronArchive/QueueConsumerTest.php

```diff
@@ -979,6 +979,55 @@ public function test_usableArchiveExists_returnsTrueIfDateRangeHasVisits_AndPeri
         $this->assertEquals([true, '2020-04-04 23:58:20'], $result);
     }
 
+    public function testUsableArchiveExistsReturnsTrueIfDateRangeHasVisitsAndPeriodIncludesTodayAndExistingPluginArchiveIsRecent()
```

Comment:
It is a very long method name. Using `@dataProvider` or creating two or more tests with shorter names might be worth it.